superset/connectors/sqla/models.py

```diff
@@ -598,18 +597,18 @@ def select_star(self) -> Optional[str]:
 
     @property
     def data(self) -> Dict[str, Any]:
-        d = super().data
+        my_data = super().data
```

Comment:
I'm not sure if `my_data` is a good variable name here, i.e., I don't understand the `my` prefix. Maybe `data_`?